drivers/network/netio/netio.c

```diff
@@ -0,0 +1,860 @@
+/*
+ * PROJECT:     NetIO driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     A more convenient networking (TCP/IP, UDP/IP) kernel API (NT6+)
+ * COPYRIGHT:   Copyright 2023-2024 Johannes Thoma <johannes@johannesthoma.com>
+ */
+
+/* We are a NT6+ driver */
+#undef _WIN32_WINNT
+#define _WIN32_WINNT 0x600
+#undef WINVER
+#define WINVER 0x600
+
+/* TODOs:
+    Done: Clean up socket in WskCloseSocket (have RefCount and
+    SocketGet / SocketPut functions).
+    Done: Remove unnecessary code (like Hook of completion)
+    Make it compile with MS VC as well
+    Rebase onto latest master
+    Squash commits (git rebase -i) one for tdihelpers one for netio
+
+    The whole Listen / Accept mechanism is still missing
+    Some minor functions (not used by WinDRBD) are missing
+    (like WskControlClient, WskSocketConnect, ...)
+    Raw sockets are not supported for now.
+    We should have regression tests for that...
+*/
+
+#include <wdm.h>
+#include <wsk.h>
+#include <ndis.h>
+#include <ntifs.h>
+#include <windef.h>
```

Comment:
windef.h has nothing to do in a NT kernel driver.

Reply:
Ok I removed it. Had to change the DWORD DebugTraceLevel (taken from afd/main.c) to int DebugTraceLevel.